### ISSUE TITLE
Compare with other NLP tools like Stanza

### ISSUE THREAD

**Issue by PoriNiki** [explosion/spaCy#5750]
Hi,
I wonder if spacy have ever compared itself with other NLP libraries and tools like [stanza](https://github.com/stanfordnlp/stanza) that recently published a [paper](https://www.aclweb.org/anthology/2020.acl-demos.14/) and said that it is far better than spacy.

**Comment by honnibal** [comment]
We're working on new benchmarks for v3, but in the meantime, the nice thing about having experiments on the same data is that not everyone needs to run every experiment.

Stanza's experiments are well-conducted overall, and their conclusion that Stanza is noticeably more accurate than spaCy is true (as of spaCy v2). There are some experiments where spaCy's score should be a few percentage points higher (e.g. by using pretrained word vectors as features, and/or by using the "spacy pretrain" feature for language model initialization). But the difference definitely wouldn't be enough to catch up and achieve similar accuracy scores to Stanza.

spaCy v3 makes it much easier to configure the models, and its machine learning API is finally well documented with the release of [Thinc](https://thinc.ai) earlier this year. I expect to release a suite of transformer-based models that achieve accuracy close to the current state of the art, perhaps even with our noses just slightly in front. (If so I'm sure someone else will publish a more accurate model shortly after.)

In the meantime, you can use the `spacy-stanza` plugin if you'd like to use Stanza with spaCy's API. Or you can just use Stanza directly --- it's up to you.

**Comment by buriy** [comment]
Actually, for Russian, using pre-trained vectors, SpaCy gave me the same quality as Stanza but SpaCy is 3x faster on CPU.
(And BERT-like models are much slower but give even higher quality).
